Assets/Scripts/UI/ToggleFullscreen.cs

```diff
@@ -0,0 +1,102 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.EventSystems;
+using System.Collections;
+
+public class ToggleFullscreen : MonoBehaviour {
+
+    public Button fullButton;
+    bool enableFull = true;
+    public Sprite OffSprite;
+    public Sprite OnSprite;
+    public GameObject Background;
+    public GameObject Fullscreen;
+
+
+    // Use this for initialization
+    void Start () {
+        //NB: 02/01/2018
+        //checks if operating system allows fullscreen and removes the button if not
+        if (SystemInfo.operatingSystem.ToLower().Contains("ios") || SystemInfo.operatingSystem.ToLower().Contains("mac")) {
+            Destroy(Background);
+            Destroy(Fullscreen);
+        }
+        Button btn = fullButton.GetComponent<Button> ();
+        btn.onClick.AddListener(FullOnClick);
```

Comment:
You aren't removing the listener anywhere in this file. Consider applying proper process here, as it's possible you can end up with a dangling listener with no object attached.